src/pages/home/home.jsx

```diff
@@ -0,0 +1,217 @@
+import "./home.css";
+
+export const Home = () => {
+  return (
+    <div>
+      <div class="home-page">
+        {/* navigation bar */}
+
+        <nav class="flex-row">
+          <div class="logo">
+            <a href="/index.html">
+              <p class="h1">SNEAKERS</p>
+            </a>
+          </div>
+
+          <div class="search-bar flex-row">
+            <input type="text" placeholder="Search" />
+            <span class="material-icons-outlined search-icon">search</span>
+          </div>
+
+          <div class="nav-items">
+            <ul class="flex-row right-icons">
+              <li>
+                <a href="/html/login.html">
```

Comment:
suggestion:you can make navbar component so your code looks more clean.

Reply:
yes, will break the homepage into several components.